README.md

```diff
@@ -2,14 +2,37 @@
 
 This is an example of an [External Issuer] for cert-manager.
 
+## Install
+
+```
+kubectl apply -f https://github.com/cert-manager/sample-external-issuer/releases/download/v0.1.0/install.yaml
+```
+
 ## Demo
 
 You can run the sample-external-issuer on a local cluster with this command:
 
 ```
-make IMG=controller:0.0.0 kind-cluster deploy-cert-manager docker-build kind-load deploy e2e
+make kind-cluster deploy-cert-manager docker-build kind-load deploy e2e
```

Comment:
The IMG value is derived from the `git describe --tags` output, so it's no longer necessary to specify it here.